src/components/MediaRenderer.tsx

```diff
@@ -7,15 +7,15 @@ interface MediaRendererProps {
     className: string;
 
     videoOveride?: ReactNode;
-    imageOveride?: ReactNode;
+    imageOverride?: ReactNode;
 }
 
 /**
  * Handles the rendering of either images or videos from a URL
  *
  * The type of rendering is infered from the file ending on the URL, the provided URL must have this file ending or the component may not work
  */
-export default function MediaRenderer({ url, className, imageOveride, videoOveride }: MediaRendererProps) {
+export default function MediaRenderer({ url, className, imageOverride: imageOveride, videoOveride }: MediaRendererProps) {
```

Comment:
refactoring react arguments is weird I know, you gotta manually rename everything
```suggestion
export default function MediaRenderer({ url, className, imageOverride, videoOveride }: MediaRendererProps) {
```

Reply:
and also update line 32